release/release.go

```diff
@@ -121,6 +122,13 @@ type dashboardReleaseNoteData struct {
 	ChangeLogData    changeLogData
 }
 
+type cliReleaseNoteData struct {
```

Comment:
This is the third struct containing the same information. Let's fine a way to create a struct that we can embed in the structs above.

Reply:
Done, let me know what you think.

1. Created a base struct:
```
type releaseNoteData struct {
	Milestone        string
	MajorMinor       string
	ChangeLogVersion string
	ChangeLogData    changeLogData
}
```
2. Removed the duplicate fields in all `*ReleaseNoteData` structs
3. Imported the `releaseNoteData` to them.

Reply:
Looks great. Thank you.